src/Datadog.Trace.ClrProfiler.Native/cor_profiler.cpp

```diff
@@ -31,9 +31,11 @@ CorProfiler::Initialize(IUnknown* cor_profiler_info_unknown) {
   const auto debug_enabled_value =
       GetEnvironmentValue(environment::debug_enabled);
 
-  if (debug_enabled_value == "1"_W || debug_enabled_value == "true"_W) {
-    debug_logging_enabled = true;
-  }
+  //if (debug_enabled_value == "1"_W || debug_enabled_value == "true"_W) {
```

Comment:
Don't forget to remove this debugging code. Thanks!

Reply:
oops :)